src/components/Menu/src/MenuOption.vue

```diff
@@ -3,6 +3,7 @@
 		:class="$s.MenuOption"
 		role="menuitem"
 		tabindex="0"
+		:style="menuOptionStyles"
```

Comment:
where is `menuOptionStyles` coming from? doesn't seem to be defined anywhere in the component

Reply:
+1